### ISSUE TITLE
Question: Why is void Write(char value) in TextWriter not abstract

### ISSUE THREAD

**Issue by roederja2** [dotnet/corefx#31777]
The comments say that the minimum a subclass must implement is void Write(char value), yet the method is virtual with an empty body instead of abstract - why?

https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/Common/src/CoreLib/System/IO/TextWriter.cs#L15

**Comment by karelz** [comment]
Good question, I expect it is some weird historical reason - we didn't make it abstract in v1 and cannot change it since then.
The class has another abstract property:
https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/Common/src/CoreLib/System/IO/TextWriter.cs#L90

**Comment by danmosemsft** [comment]
@roederja2 Yes, it would break everyone deriving from this class who did not override it. We can't do that for something as core as TextWriter.

Is there something you are trying to achieve that you can't? Or just a cleanliness observation? If the latter I think we can close as it's breaking to fix.
